### ISSUE TITLE
[Install] ImportError: cannot import name 'validate_email_add' from 'frappe.utils'

### ISSUE THREAD

**Issue by tr37ion** [frappe/erpnext#16939]
Trying to install ERPnext following:
```
$ git clone https://github.com/frappe/bench bench-repo
$ pip2 install --user -e bench-repo
$ echo "export PATH=$HOME/.local/bin:$PATH" >> ~/.bashrc
$ source ~/.bashrc
$ bench init frappe-bench && cd frappe-bench
$ bench get-app erpnext https://github.com/frappe/erpnext
$ bench new-site site1.local
$ bench --site site1.local install-app erpnext
$ bench use site1.local
$ bench start
```
Unfortunately, I always get the following error messages:
```
$ bench new-site site1.local

MySQL root password:

Installing frappe...
Updating DocTypes for frappe        : [========================================]
Updating country info               : [========================================]
Set Administrator password:
Re-enter Administrator password:
*** Scheduler is disabled ***
[erp@host frappe-bench]$ bench --site site1.local install-app erpnext

Installing erpnext...
Updating DocTypes for erpnext       : [=                                       ]Traceback (most recent call last):
  File "/home/erp/erpnext/frappe-bench/apps/frappe/frappe/modules/utils.py", line 199, in load_doctype_module
    doctype_python_modules[key] = frappe.get_module(module_name)
  File "/home/erp/erpnext/frappe-bench/apps/frappe/frappe/__init__.py", line 799, in get_module
    return importlib.import_module(modulename)
  File "/home/erp/erpnext/frappe-bench/env/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/erp/erpnext/frappe-bench/apps/erpnext/erpnext/accounts/doctype/payment_entry/payment_entry.py", line 11, in <module>
    from erpnext.accounts.doctype.journal_entry.journal_entry import get_default_bank_cash_account
  File "/home/erp/erpnext/frappe-bench/apps/erpnext/erpnext/accounts/doctype/journal_entry/journal_entry.py", line 11, in <module>
    from erpnext.hr.doctype.expense_claim.expense_claim import update_reimbursed_amount
  File "/home/erp/erpnext/frappe-bench/apps/erpnext/erpnext/hr/doctype/expense_claim/expense_claim.py", line 9, in <module>
    from erpnext.hr.utils import set_employee_name
  File "/home/erp/erpnext/frappe-bench/apps/erpnext/erpnext/hr/utils.py", line 10, in <module>
    from erpnext.hr.doctype.employee.employee import get_holiday_list_for_employee
  File "/home/erp/erpnext/frappe-bench/apps/erpnext/erpnext/hr/doctype/employee/employee.py", line 7, in <module>
    from frappe.utils import getdate, validate_email_add, today, add_years, format_datetime
ImportError: cannot import name 'validate_email_add' from 'frappe.utils' (/home/erp/erpnext/frappe-bench/apps/frappe/frappe/utils/__init__.py)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib64/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib64/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/erp/erpnext/frappe-bench/apps/frappe/frappe/utils/bench_helper.py", line 97, in <module>
    main()
  File "/home/erp/erpnext/frappe-bench/apps/frappe/frappe/utils/bench_helper.py", line 18, in main
    click.Group(commands=commands)(prog_name='bench')
  File "/home/erp/erpnext/frappe-bench/env/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/erp/erpnext/frappe-bench/env/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invarcherpnextoke(ctx)
  File "/home/erp/erpnext/frappe-bench/env/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/erp/erpnext/frappe-bench/env/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/erp/erpnext/frappe-bench/env/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/erp/erpnext/frappe-bench/env/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/erp/erpnext/frappe-bench/env/lib/python3.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/erp/erpnext/frappe-bench/apps/frappe/frappe/commands/__init__.py", line 25, in _func
    ret = f(frappe._dict(ctx.obj), *args, **kwargs)
  File "/home/erp/erpnext/frappe-bench/apps/frappe/frappe/commands/site.py", line 169, in install_app
    _install_app(app, verbose=context.verbose)
  File "/home/erp/erpnext/frappe-bench/apps/frappe/frappe/installer.py", line 84, in install_app
    sync_for(name, force=True, sync_everything=True, verbose=verbose, reset_permissions=True)
  File "/home/erp/erpnext/frappe-bench/apps/frappe/frappe/model/sync.py", line 56, in sync_for
    reset_permissions=reset_permissions, for_sync=True)
  File "/home/erp/erpnext/frappe-bench/apps/frappe/frappe/modules/import_file.py", line 65, in import_file_by_path
    ignore_version=ignore_version, reset_permissions=reset_permissions)
  File "/home/erp/erpnext/frappe-bench/apps/frappe/frappe/modules/import_file.py", line 133, in import_doc
    doc.insert()
  File "/home/erp/erpnext/frappe-bench/apps/frappe/frappe/model/document.py", line 249, in insert
    self.run_post_save_methods()
  File "/home/erp/erpnext/frappe-bench/apps/frappe/frappe/model/document.py", line 916, in run_post_save_methods
    self.run_method("on_update")
  File "/home/erp/erpnext/frappe-bench/apps/frappe/frappe/model/document.py", line 781, in run_method
    out = Document.hook(fn)(self, *args, **kwargs)
  File "/home/erp/erpnext/frappe-bench/apps/frappe/frappe/model/document.py", line 1061, in composer
    return composed(self, method, *args, **kwargs)
  File "/home/erp/erpnext/frappe-bench/apps/frappe/frappe/model/document.py", line 1044, in runner
    add_to_return_value(self, fn(self, *args, **kwargs))
  File "/home/erp/erpnext/frappe-bench/apps/frappe/frappe/model/document.py", line 775, in <lambda>
    fn = lambda self, *args, **kwargs: getattr(self, method)(*args, **kwargs)
  File "/home/erp/erpnext/frappe-bench/apps/frappe/frappe/core/doctype/doctype/doctype.py", line 264, in on_update
    self.run_module_method("on_doctype_update")
  File "/home/erp/erpnext/frappe-bench/apps/frappe/frappe/core/doctype/doctype/doctype.py", line 331, in run_module_method
    module = load_doctype_module(self.name, self.module)
  File "/home/erp/erpnext/frappe-bench/apps/frappe/frappe/modules/utils.py", line 201, in load_doctype_module
    raise ImportError('Module import failed for {0} ({1})'.format(doctype, module_name + ' Error: ' + str(e)))
ImportError: Module import failed for Payment Entry (erpnext.accounts.doctype.payment_entry.payment_entry Error: cannot import name 'validate_email_add' from 'frappe.utils' (/h
ome/erp/erpnext/frappe-bench/apps/frappe/frappe/utils/__init__.py))
```
This is not related to #16929 or?

**Comment by cjpit** [comment]
It looks like the same issue as you have linked, can we get the pr pushed through to develop asap?

**Comment by cjpit** [comment]
Yay sorry this has been merged :)

**Comment by Mangesh-Khairnar** [comment]
This issue has been fixed #16929
